src/backend/dx11/src/lib.rs

```diff
@@ -1840,8 +1848,12 @@ pub struct PipelineLayout {
 #[derive(Debug)]
 pub struct DescriptorSetLayout {
     bindings: Vec<PipelineBinding>,
-    offset_mapping: Vec<(u32, pso::DescriptorType)>,
-    handle_count: u32
+    register_remap: Vec<(pso::DescriptorType, u32, u8, bool)>,
+    handle_count: u32,
+    num_s: u8,
```

Comment:
sounds like we need a struct with those - would also help sharing the code collecting the counters